hack/check-file-length.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash -ex
+
+filename=$1
+if [ -z "$filename" ]; then
+    echo >&2 $0 expects a filename
+    exit 1
+fi
+
+max=$(grep lint_maximum_file_length "$filename" | awk '{print $3}')
+len=$(wc -l < "$filename")
+
+if [ -z "$max" ]; then
+    echo >&2 $filename: maximum length not found
+    exit 1
+fi
+
+if [ "$len" -gt "$max" ]; then \
+    echo >&2 $filename: file longer than $max
+    exit 1
+fi
+
+if [ "$len" -lt $[max-100] ]; then \
```

Comment:
Is it necessary to enforce the lower bound as well?

Reply:
I think that it can serve as a reminder to code authors that it probably time to tighten our limit. Do you see a problem with it? Do you prefer a monthly review to tighten the limit?

Reply:
I'd say it's better to be reminded by the check, instead of doing so manually. All good